apps/api/src/modules/stores/stores.service.ts

```diff
@@ -1,56 +1,88 @@
 import { CreateStoreDto, UpdateStoreDto } from '@meal-time/api-interfaces';
 import {
-  ForbiddenException,
-  Injectable,
-  NotFoundException,
+	ForbiddenException,
+	Injectable,
+	NotFoundException,
 } from '@nestjs/common';
 import { PrismaClientKnownRequestError } from '@prisma/client/runtime';
 import { PrismaService } from '../../services/prisma/prisma.service';
 import { STORES_EXCEPTION_MSG } from './stores.exceptionMessages';
 
 @Injectable()
 export class StoresService {
-  constructor(private prisma: PrismaService) {}
-
-  async create(createStoreDto: CreateStoreDto) {
-    return this.prisma.stores
-      .create({
-        data: {
-          ...createStoreDto,
-        },
-      })
-      .catch((error: PrismaClientKnownRequestError) => {
-        if (error.code === 'P2002') {
-          throw new ForbiddenException(STORES_EXCEPTION_MSG.ALREADY_EXISTS);
-        }
-      });
-  }
-
-  findAll() {
-    return `This action returns all stores`;
-  }
-
-  findOne(id: number) {
-    return `This action returns a #${id} store`;
-  }
-
-  update(id: number, updateStoreDto: UpdateStoreDto) {
-    return `This action updates a #${id} store`;
-  }
-  
-  async remove(id: number) {
-    const store = await this.prisma.stores.findUnique({
-      where: {
-        id,
-      },
-    });
-
-    if (!store) throw new NotFoundException(STORES_EXCEPTION_MSG.NOT_FOUND);
-
-    return this.prisma.stores.delete({
-      where: {
-        id,
-      },
-    });
-  }
+	constructor(private prisma: PrismaService) {}
+
+	async create(createStoreDto: CreateStoreDto) {
+		return this.prisma.stores
+			.create({
+				data: {
+					...createStoreDto,
+				},
+			})
+			.catch((error: PrismaClientKnownRequestError) => {
+				if (error.code === 'P2002') {
+					throw new ForbiddenException(
+						STORES_EXCEPTION_MSG.ALREADY_EXISTS
+					);
+				}
+			});
+	}
+
+	async findAll() {
+		const stores = await this.prisma.stores.findMany();
+		// Sorting the array of stores in ascending order
+		await stores.sort((a, b) => {
+			if (a.id > b.id) {
+				return 1;
+			} else if (a.id < b.id) {
+				return -1;
+			}
+			return 0;
+		});
```

Comment:
You should just be able to sort in ascending order by

```ts
const stores = await this.prisma.stores.findMany({
  orderBy: {
    author: {
      id: 'asc',
    },
  },
});
```

From: https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting#sort-by-relation

Reply:
Bruh kiggede derinde i går og kunne ikke se der.... I will change

Reply:
I actually looked this up yesterday but could not see it. Thanks.